### ISSUE TITLE
search functionality

### ISSUE THREAD

**Issue by alloursjournall** [frankcollins3/mine_nugget#4]
![search functionality](https://user-images.githubusercontent.com/73137934/179611985-5c7d1b01-54b0-48cc-a264-f8a4edaf7337.png)

  inp.on('search', (event) => {
        $('.searchGrid').children().detach()         // this is our container.truncate() method.
        let searchword = $(event.target).val() // kind of wanted [.target.val().UpperCase()]
        console.log($('.liStrain'))       
        $('.liStrain').each( (i, strain) => {
            // console.log($(strain).text().charAt(1)) // not array based indexing. 
            let firstLetter = $(strain).text().charAt(1)
            if (firstLetter == searchword) {     // i wonder why strict equality only returns 1 value when other dataNames satisfy the condition of ".charAt(1) == 'w' for example.". 
                console.log("heres the text")
                console.log($(strain).text())
                let searchStrainText = $(strain).text()

                let searchMine = document.createElement('div')
                $(searchMine)
                .css('height', '45px')
                .css('width', '45px')
                .css('border', '2px solid papayawhip')
                .css('border-radius', '50%')          
                .css('background-image', `url('/img/mine.png')`)
                .css('background-size', 'cover')
                .css('background-repeat', 'no-repeat')
                .css('margin-top', '1em')
                $('.searchGrid').append($(searchMine))

                let searchMineLabel = document.createElement('p')
                $(searchMineLabel)
                .css('color', 'papyawhip')
                .css('font-weight', 'bold')
                .html(searchStrainText)
                // .attr('for', 'div')
                // .html( $(strain).text())
                
                $(searchMineLabel).insertAfter($(searchMine))
                // console.log($(this))
                // console.log($(this).length)
            }
        })
    })
